function resolve(string) {
  try {return eval(string)}
  catch {return 'Error'}
}